libs/qCC_db/include/ccColorTypes.h

```diff
@@ -244,6 +244,12 @@ namespace ccColor
 					static_cast<ColorCompType>(color.b * MAX) );
 	}
 	
+	//! Conversion from Rgb to Rgba
+	inline Rgba FromRgbToRgba(const Rgb& color)
```

Comment:
Mind that there's already a constructor of Rgba that takes a Rgb instance as first arugment, and an alpha value as second argument. --> return Rgba(color, MAX);

Reply:
Fair enough, I didn't notice it I was looking for a conversion function following the pattern of the others, I don't care much either way.

Reply:
Well you can keep the more explicit conversion function. It's just to make the 'return' line more compact and efficient.